### ISSUE TITLE
XVG transfer from electrum wallet to Binance stuck with 0 confirmations for several days.

### ISSUE THREAD

**Issue by smehltrett** [vergecurrency/VERGE#489]
# Issue Template

## Context

I transferred some xvg from my local electrum wallet to binance. It has been stuck with 0 confirmation since January 12th and binance says the issue is not on their end they cannot help me.

Transaction ID: a53a50c4d5bdc1cdfe744a48c80f6d2bacfa24b4b7d95cc6e0c7e2baa7c0605d
Link to the blockchain explorer
[https://verge-blockchain.info/tx/a53a50c4d5bdc1cdfe744a48c80f6d2bacfa24b4b7d95cc6e0c7e2baa7c0605d](url)

## Prerequisites

* [x] Are you running the latest version? 

No I am running electrum 2.4.1

## User Story:

* "I'm submitting a ... " Support Request
  - [ ] bug report
  - [ ] feature request
  - [ X] support request 
  - [ ] translation request 

## Your Environment

* Verge Release: Electrum - XVG 2.4.1 
* CPU: i7 8700k
* GPU: Gtx 1080
* RAM: 16gb
* OS: Windows 10
    - [ ] I'm using Mac
    - [ ] I'm using Linux
    - [ ] I'm using Mac

**Comment by benammann** [comment]
already known issue, they are working on this.

I think at block height 1807000 this will be fixed ~ 24 hours

**Comment by smehltrett** [comment]
so it will eventually go through I don't have to do anything?

**Comment by Gallmore** [comment]
I'm having exactly the same problem. Running electrum 2.4.1.
It could be related to 'set transaction fee manually' option in the tools/preferences, which I could've used.

**Comment by ryanprovofc** [comment]
Having this issue as well with the electrum 2.4.1 wallet and been hearing it more frequently lately.  Hopefully somebody will be able to respond with a solution.  I've been stuck for about 5 days now.  Don't want to start deleting and re-installing the wallet just yet.  Would rather wait it out till I hear something.

**Comment by EXAIYU** [comment]
Already started deleting and reinstalling ...
doesn't work.

**Comment by smehltrett** [comment]
no official response from team :( this seems like a major issue....

**Comment by caniyabanci76** [comment]
same problem here....

**Comment by jrsikors** [comment]
did the hard fork resolve these issues?

**Comment by Influence2** [comment]
Were these issues resolved after hardfork as stated?

**Comment by jrsikors** [comment]
@justinvforvendetta  close this one out plz

**Comment by jrsikors** [comment]
@justinvforvendetta nudge

**Comment by Glebbas** [comment]
I've been waiting for an update to Electrum, which would fix the unverified coins problem, but even now after the fork, it hasn't started working and there is no updates to the Electrum  wallet. My coins are locked in the wallet. Even the screenshot shows Electrum is disconnected, it connects every now and then, but the status won't change.

![xvg1](https://user-images.githubusercontent.com/36762706/36592190-651bfa6c-189d-11e8-833f-f513b47c0b7a.jpg)

![xvg2](https://user-images.githubusercontent.com/36762706/36592191-653cabd6-189d-11e8-90d6-5aa93c35b072.jpg)

**Comment by jrsikors** [comment]
You don't need to hide the TX's and addresses and stuff, that stuff actually helps us troubleshoot better.

Have you tried 'uninstalling' the electrum XVG wallet and re-importing your wallet address?

**Comment by jrsikors** [comment]
do a wallet -> private key -> export ... i would advise not saving it as a CSV .. just grab the private key of the address you want (probably just DDPVCBroBkeWaZnJyJ8fiwhsaA6gniT3vR ? )

close down electrum
go to c:\users\<YOU>\AppData\Roaming\Electrum-XVG\wallets

back up the file first, copy it to a jump drive or somewhere safe

then rename it to like default_wallet_old

then re-start electrum-xvg... should ask if you want to recover a wallet or create new one .. and go from there.  Either create a new wallet and import your private key... 

Bottom line is...
Your transaction isn't stuck on the blockchain, it's stuck on Electrum.  You can easily fix this yourself by importing your private key somewhere else.  Either QT wallet.. or a new Electrum wallet.

**Comment by jrsikors** [comment]
Also --- check this one out :
https://www.reddit.com/r/vergecurrency/comments/7uelnl/electrum_wallet_offline_please_help/

There's a link to a youtube video that looks good

**Comment by jrsikors** [comment]
Nutshell instructions:
Step 1) Close down Electrum wallet
Step 2) Browse to where your Electrum 'installation' is
Step 3) Go into the Lib folder
Step 4) Open network.py in notepad or textpad , some kind of text editor
Step 5) 
Replace this chunk of code:
```
DEFAULT_SERVERS = {
    'electrum-verge.xyz':{'t':'50001', 's':'50002'},
    'electrum-xvg.stream':{'t':'50001', 's':'50002'},
    'electrum-xvg.party':{'t':'50001', 's':'50002'}
}
```

With this chunk of code:
```
DEFAULT_SERVERS = {
    'electrum-verge.xyz':{'t':'50001', 's':'50002'},
    'electrum-xvg.stream':{'t':'50001', 's':'50002'},
    'electrum-xvg.party':{'t':'50001', 's':'50002'},
    'e1.verge-electrum.com':{'t':'50001', 's':'50002'},
    'e2.verge-electrum.com':{'t':'50001', 's':'50002'},
    'e3.verge-electrum.com':{'t':'50001', 's':'50002'}
}
```

step 6)  Open Electrum
step 7)  Go to tools -> network
step 8)  Make sure "Auto-connect" is checked, and the 3 electrum servers are listed:
   e1.verge-electrum.com
   e2.verge-electrum.com
   e3.verge-electrum.com
